osquery/tables/applications/windows/carbon_black.cpp

```diff
@@ -26,17 +28,18 @@ namespace tables {
 
 // Carbon Black registry path
 #define kCbRegLoc "SOFTWARE\\CarbonBlack\\config"
-// Path to Carbon Black direcotry
-#define kCbDir "C:\\Windows\\CarbonBlack\\"
 
 void getQueue(Row& r) {
+  char windowsPath[MAX_PATH];
+  GetWindowsDirectory(windowsPath, MAX_PATH);
```

Comment:
I have an PR inbound, that contains a new fn in filesystem, `getSystemRoot()` that will return an fs::path.